src/app/personal-info/personal-info.component.ts

```diff
@@ -457,4 +483,18 @@ this.showLastTax = true;
 
 }
 
+//https://s3.amazonaws.com/digitaxbucket/1603178195073-all_data.zip

```

Comment:
@Bithika-Redapple If you want to make a comment regarding the download link, form a full comment. For example,

"Method is used to download tax information from s3. The download link is, https://s3.amazonaws.com/digitaxbucket/1603178195073-all_data.zip"

Reply:
modified